edit-post/components/plugin-sidebar/index.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, Children, cloneElement, compose } from '@wordpress/element';
+import { Slot, Fill, withFocusReturn } from '@wordpress/components';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { withPluginContext } from '../../api/plugin';
+import SidebarLayout from './sidebar-layout';
+
+/**
+ * Name of slot in which popover should fill.
+ *
+ * @type {String}
+ */
+const SLOT_NAME = 'PluginSidebar';
+
+class SidebarErrorBoundary extends Component {
```

Comment:
This component should be put in its own file. In the long run, it should be generalized to support or types of plugins.